packaging/packager

```diff
@@ -56,7 +56,7 @@ if ! type zip > /dev/null 2>&1; then
     exit 1
 fi
 function package_libc_via_pacman {
-    if grep "Arch Linux" < /etc/os-release > /dev/null 2>&1; then
+    if grep -E "Arch Linux|Manjaro Linux" < /etc/os-release > /dev/null 2>&1; then
```

Comment:
Use the full option for readability please.
`--extended-regexp` instead of `-E`

Reply:
Thanks for the review @marcomagdy. Now updated to use `--extended-regexp`. Please re-review when you can, thanks